### PR TITLE
[multi-lang-privacy] Allow loading multi lang privacy

### PR DESCRIPTION
**Pull request by inverse**

### CODE REVIEW

File: src/startup/startup.js [hunk 1]
```diff
@@ -26,15 +26,36 @@ require([
...
+	const locale = window.navigator.language.split('-')[0];
+	const defaultPrivacyDoc = 'PRIVACY.md';
 
-	fetch(privacyUrl)
+	let privacyDoc = defaultPrivacyDoc;
+	if (locale !== 'en') {
+		privacyDoc = `PRIVACY.${locale}.md`;
+	}
+
+	fetch(chrome.runtime.getURL(privacyDoc))
```
Comment (inverse) [inline src/startup/startup.js]: Is it worth splitting this out into some md loader class?

Comment (alexesprit) [reply src/startup/startup.js]: No, but I'd use for..of loop and async/await instead to simplify the logic and code. Something like this:
```js
let privacyDoc = `PRIVACY.__MSG_@@ui_locale__.md`;

let langFiles = [defaultPrivacyDoc];
if (locale !== 'en') {
	// Insert translated MD file, so we can try to use it first
	languages.unshift(privacyDoc);
}

for (let f of langFiles) {
    // Fetch language; if everything is ok, break the loop
}

File: src/startup/startup.js [hunk 2]
```diff
@@ -26,15 +26,36 @@ require([
 		$('.finished').show();
 	};
 
-	let privacyUrl = chrome.runtime.getURL('PRIVACY.md');
+	const locale = 'riu'; // window.navigator.language.split('-')[0];
```
[the remark this answers is not in the document]
Comment (inverse) [reply src/startup/startup.js]: Whats wrong with using `window.navigator.language.split('-')[0]`?

File: src/startup/startup.js (hunk 1 above)
Comment (inverse) [reply src/startup/startup.js]: Updated

**Comment by inverse** [comment]
@alexesprit updated :+1:

**Pull request merged**